src/sardana/macroserver/macros/scan.py

```diff
@@ -1848,3 +1849,78 @@ def getTimeEstimation(self):
 
     def getIntervalEstimation(self):
         return self.nr_interv
+
+
+class scanstats(Macro):
+    """Calculate basic statistics of the first enabled and plotted counter in
+    the active measurement group for the last scan. Print it and publish it
+    in the env. The macro must be hooked in the post-scan hook place.
+    """
+
+    def run(self, *args):
+        parent = self.getParentMacro()
+        if parent:
+            mntGrp = self.getEnv('ActiveMntGrp')
+            self.mntGrp = self.getObj(mntGrp, type_class=Type.MeasurementGroup)
+            channels = self.mntGrp.getChannels()
+            select_channel = ''
```

Comment:
I think that initialization with `None` and comparison with `None` is more pythonic.